pelican/tests/test_contents.py

```diff
@@ -341,23 +342,25 @@ def test_intrasite_link_more(self):
         cls_name = '_DummyAsset' if six.PY3 else b'_DummyAsset'
 
         args = self.page_kwargs.copy()
-        args['settings'] = get_settings()
+        args['settings'] = get_settings(STATIC_PATHS=[])
         args['source_path'] = 'content'
-        args['context']['filenames'] = {
-            'images/poster.jpg': type(
-                cls_name, (object,), {'url': 'images/poster.jpg'}),
-            'assets/video.mp4': type(
-                cls_name, (object,), {'url': 'assets/video.mp4'}),
-            'images/graph.svg': type(
-                cls_name, (object,), {'url': 'images/graph.svg'}),
-            'reference.rst': type(
-                cls_name, (object,), {'url': 'reference.html'}),
+        args['context']['static_content'] = {
+            'images/poster.jpg':
+                type(cls_name, (object,), {'url': 'images/poster.jpg'}),
+            'assets/video.mp4':
+                type(cls_name, (object,), {'url': 'assets/video.mp4'}),
+            'images/graph.svg':
+                type(cls_name, (object,), {'url': 'images/graph.svg'}),
+        }
+        args['context']['generated_content'] = {
+            'reference.rst':
+                type(cls_name, (object,), {'url': 'reference.html'}),
         }
 
         # video.poster
         args['content'] = (
             'There is a video with poster '
-            '<video controls poster="{filename}/images/poster.jpg">'
+            '<video controls poster="{static}/images/poster.jpg">'
             '<source src="|filename|/assets/video.mp4" type="video/mp4">'
```

Comment:
This should be `|static|` as well.

Reply:
Thanks, I've replaced a bunch of other occurrences of `|filename|` by `|static|` in the test material.